server/webapp/WEB-INF/rails.new/app/assets/javascripts/vsm_renderer.js

```diff
@@ -321,13 +321,16 @@ Graph_Renderer = function (container) {
         gui += '<li class="instance">';
         if (instance.label != '') {
             if (isCurrent) {
-                gui += '<h4 title="' + instance.label + '"><i class="label">Label: </i>' + instance.label + '</h4>';
+                gui += '<h4 title="' + instance.label + '"><span class="pipeline_run_label">Instance:' + instance.label + '</span></h4>';
```

Comment:
Could you write javascript tests to get this duration verified for different cases?